sdk/eventhub/event-hubs/README.md

```diff
@@ -237,28 +239,26 @@ To stop receiving events, you can call `close()` on the object returned by the `
 ```javascript
 const { EventHubConsumerClient } = require("@azure/event-hubs");
 const { ContainerClient } = require("@azure/storage-blob");
-const { BlobPartitionManager } = require("@azure/eventhubs-checkpointstore-blob");
+const { BlobCheckpointStore } = require("@azure/eventhubs-checkpointstore-blob");
 
 async function main() {
   const consumerClient = new EventHubConsumerClient("my-consumer-group", "connectionString", "eventHubName");
   const blobContainerClient = new ContainerClient("storage-connection-string", "container-name");
   await blobContainerClient.create(); // This can be skipped if the container already exists
-  const partitionManager =  new BlobPartitionManager(blobContainerClient);
-
-  const myEventHandler = (events, context) => {
-    // your code here
-  };
-  const myErrorHandler = (err, context) => {
-    // your error handling code here
-  };
-  const subscription = consumer.subscribe(partitionManager, {
-    processEvents: myEventHandler,
-    processError: myErrorHandler
+  const partitionManager =  new BlobCheckpointStore(blobContainerClient);
```

Comment:
```suggestion
  const checkpointStore=  new BlobCheckpointStore(blobContainerClient);
```

Reply:
Had to fix even more than that because this one was also using the wrong constructor and still passing the partition manager to subscribe.